### PR TITLE
Fix gecko

### PR DESCRIPTION
**Pull request by HauntedChest13**
tails review thankssks

### CODE REVIEW

File: src/main/java/com/crypticcosmos/crypticcosmos/creatures/moon_frog/MoonFrogModel.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        return new ResourceLocation(CrypticCosmos.MOD_ID, "animations/moonfrog.json");
+    }
+
+    @Override
+    public void setLivingAnimations(MoonFrogEntity entity, Integer uniqueID, @Nullable AnimationEvent customPredicate) {
+        super.setLivingAnimations(entity, uniqueID, customPredicate);
+        EntityModelData data = (EntityModelData) customPredicate.getExtraData().get(0);
+        // Apply head look to model
+        IBone head = this.getAnimationProcessor().getBone("Head");
+        head.setRotationY((float) Math.toRadians(Utils.clamp(data.netHeadYaw, -45, 45)));
```
Comment (TheOnlyTails) [inline src/main/java/com/crypticcosmos/crypticcosmos/creatures/moon_frog/MoonFrogModel.java:39]: > `Utils.clamp`

Where does this method come from?

**Review by TheOnlyTails — CHANGES_REQUESTED** [review]
LGTM! Just fix that one little thing, and add the moon beast fixes to here as well.

**Comment by HauntedChest13** [comment]
So the head turns on the moon frog randomly so it looks like a mob smh

**Comment by HauntedChest13** [comment]
Go check the moon frog model
its all there

**Comment by TheOnlyTails** [comment]
I checked it out, but that method doesn't exist.

**Comment by TheOnlyTails** [comment]
I'll just replace it with `MathUtils.clamp`.

**Comment by HauntedChest13** [comment]
no dont it works really well.

**Comment by TheOnlyTails** [comment]
But.... `Utils` doesn't exist!

**Comment by HauntedChest13** [comment]
it does now 😏

**Comment by TheOnlyTails** [comment]
btw, just so you know, writing "Closes #issuenumber" at the end of a PR automatically links them, so you can close the issue automatically when the PR is merged.

**Review by TheOnlyTails — APPROVED** [review]
LGTM!

**Comment by TheOnlyTails** [comment]
Now that I approved, once the check passes, this will be merged, the issue closed, and the card moved to "Done".

**Comment by HauntedChest13** [comment]
Alrighty, cool!

**Pull request merged**